internal/model/Firm.go

```diff
@@ -0,0 +1,15 @@
+package model
+
+import "fmt"
+
+type Firm struct {
+	Id     int    `json:"id"`
+	Name   string `json:"firmName"`
+	Number int    `json:"firmNumber"`
+}
+
+func (f Firm) GetFirmURL() string {
+	url := "/supervision/deputies/firm/%d"
+
+	return fmt.Sprintf(url, f.Id)
```

Comment:
Maybe we should have a test for this, just to ensure it doesn't get broken by something in the future?

Reply:
It's under firm_test I'm afraid

Reply:
Oops!